spec/models/ability_spec.rb

```diff
@@ -302,4 +302,21 @@
     expect(ability).to be_able_to(:send_email, Email)
   end
 
+
+  %i[pupil coach].each do |role|
```

Comment:
Now as review: 
These tests are already in the file, adding them would mean we have these tests 2-times